lib/customisation.nix

```
@@ -66,6 +66,56 @@ rec {
         __spliced = {} // (mapAttrs (_: sDrv: overrideDerivation sDrv f) drv.__spliced);
       });
 
+  makeOverridableGeneric =
+    {
+      name,
+      inner,
+      wrap ? [],
```

Does this argument enables wrapping existing overriders such as `overrideAttrs` and `override`? If so, how about giving it some sensible default elements`?
```suggestion
      wrap ? [ "override" "overrideAttrs" "overrideDerivation" ],
```